src/testing/vortex/supervisor.zig

```diff
@@ -459,7 +437,7 @@ const Supervisor = struct {
                     std.posix.SIG.KILL => log.info("workload terminated as requested", .{}),
                     else => {
                         log.err("workload exited unexpectedly with signal {d}", .{signal});
-                        std.process.exit(1);
+                        return error.TestFailed;
```

Comment:
If we fail via `process.exit(1)` then the `defer`s aren't called, so the vortex's tmp directory isn't cleaned up.

Reply:
drive by comment, but a nice strategy might be to cleanup directory on startup, rather than on exit. Unconditionally leaving data files/logs around for inspection might be useful! Though this requires assuming that there's at most one Vortex running at a time in a given directory, this might be a reasonable thing to do? Vortex isn't featherweight, given that it spawns real tiger beetles!